catalog/spatial/wfs/spatial-wfs-converter/src/main/java/org/codice/ddf/spatial/ogc/wfs/catalog/converter/impl/AbstractFeatureConverter.java

```diff
@@ -307,6 +322,26 @@ protected Serializable getValueForMetacardAttribute(AttributeFormat attributeFor
 
     }
 
+    private Geometry transformToWG84(Geometry geometry) {
```

Comment:
I think it would be useful to be consistent here with the CRS. Instead of `transformWGS84()` I think `transformEPSG4326` makes more sense. (Nit picky I know, but some people may not know they are effectively the same)